helm/dashboards/dashboards/shared/private/loki-cost-estimation.json

```diff
@@ -1097,7 +1096,7 @@
     ]
   },
   "time": {
-    "from": "now-1h",
+    "from": "now-7d",
```

Comment:
Is 7-days the expected default timeframe now?

Reply:
No, just an error from my side